lib/matchers/common-types.ts

```diff
@@ -67,5 +67,80 @@ export default () => {
         pass,
       };
     },
+    /**
+     * Checks if the actual value is equal to the expected value for a given field.
+     * @template T - The type of the field.
+     * @param {T & State<T>} actual - The actual field value to compare.
+     * @param {PrivateKey} value - The expected field value to compare.
+     * @returns {MatcherResult} - The result of the comparison.
+     */
+    toEqualPrivateKey<T extends PrivateKey>(actual: T & State<T>, value: PrivateKey) {
```

Comment:
Does it make sense to store in the contract state the private key? 🤔

Reply:
It doesn't. Removed.